spec/factories/claims.rb

```diff
@@ -68,5 +68,10 @@
 
       eligibility_factory { ["#{policy.to_s.underscore}_eligibility".to_sym, :ineligible] }
     end
+
+    trait :pii_removed do
```

Comment:
To make this trait realistic, can we `nil` out the details that should be blank?